php/composer.json

```
@@ -3,6 +3,9 @@
     "description": "Std UriTemplate, RFC 6570 implementation",
     "type": "library",
     "license": "Apache-2.0",
+    "require": {
+        "php": ">=7.4"
```

```suggestion
        "php": "^7.4 || ^8.0"
```
To protect us from breaking changes post PHP 8.x